install/run.php

```diff
@@ -297,7 +297,7 @@ function init_read_log() {
 
         return new Promise((resolve, reject) => {
             $.ajax({
-                url: "<?php echo $_POST['websiteurl'] ?? $websiteurl; ?>" + "index.php/update/dxcc",
+                url: url: `${window.location.origin}/index.php/update/dxcc`,
```

Comment:
Remove that c+p mistake :)
```suggestion
                url: `${window.location.origin}/index.php/update/dxcc`,
```

Reply:
Doh, that's embarrassing. Nice catch!